### DISCUSSION TITLE
Does Kuzu have a memory only mode?

### DISCUSSION THREAD

**Discussion by cmcclung-ANGENG** [kuzudb/kuzu#3500] [Q&A] [answered]
Hi,

Has there been any prior discussion about making Kuzu not require a db file?

It'd be kindof useful to ensure data doesn't exist after the app is stopped etc.

**Comment by prrao87** [comment, marked as answer]
This is ~~an open~~ a resolved issue in v0.6.0, see https://github.com/kuzudb/kuzu/issues/1816 and https://github.com/kuzudb/kuzu/pull/4012.

**Comment by mewim** [comment]
In-memory mode has been added in https://github.com/kuzudb/kuzu/pull/4012 and now available with [release v0.6.0](https://github.com/kuzudb/kuzu/releases/tag/v0.6.0).
